{
  "id": "12342992",
  "name": "10.14.2.0",
  "description": "",
  "date": "2018-05-03",
  "version": "10.14.2.0",
  "module": "",
  "lastUpdated": 1633057036661,
  "type": "release",
  "_links": {
    "self": {
      "href": "https://apis.danklco.com/apache-release-info/DERBY/12342992"
    },
    "jira": {
      "href": "https://issues.apache.org/jira/browse/DERBY/fixforversion/12342992"
    },
    "project": {
      "href": "https://apis.danklco.com/apache-release-info/DERBY"
    }
  },
  "releaseNotes": "Release Notes - Derby - Version 10.14.2.0",
  "_embedded": {
    "issues": [
      {
        "issue": "DERBY-6986",
        "title": "Network Server COMMAND_TESTCONNECTION need not try to open a database",
        "type": "Bug",
        "_links": {
          "release": {
            "href": "https://apis.danklco.com/apache-release-info/DERBY/12342992"
          },
          "jira": {
            "href": "https://issues.apache.org/jira/browse/DERBY-6986"
          }
        }
      },
      {
        "issue": "DERBY-6987",
        "title": "The default Network Server security policy file could be trimmed down somewhat.",
        "type": "Bug",
        "_links": {
          "release": {
            "href": "https://apis.danklco.com/apache-release-info/DERBY/12342992"
          },
          "jira": {
            "href": "https://issues.apache.org/jira/browse/DERBY-6987"
          }
        }
      },
      {
        "issue": "DERBY-7022",
        "title": "I have a table with autonumeric field, this is the primary key, when I try to import with SYSCS_UTIL.SYSCS_IMPORT_TABLE_BULK give error  Attempt to modify an identity column 'IDTIPOITEM'.' was thrown while evaluating an expression.",
        "type": "Bug",
        "_links": {
          "release": {
            "href": "https://apis.danklco.com/apache-release-info/DERBY/12342992"
          },
          "jira": {
            "href": "https://issues.apache.org/jira/browse/DERBY-7022"
          }
        }
      },
      {
        "issue": "DERBY-7098",
        "title": "Derby issues an ERROR  java.sql.SQLException: ResultSet not open. Operation 'next' not permitted. Verify that autocommit is off",
        "type": "Bug",
        "_links": {
          "release": {
            "href": "https://apis.danklco.com/apache-release-info/DERBY/12342992"
          },
          "jira": {
            "href": "https://issues.apache.org/jira/browse/DERBY-7098"
          }
        }
      },
      {
        "issue": "DERBY-7118",
        "title": "Derby with hibernate does not generate relationship mappings in *hbm.xml",
        "type": "Bug",
        "_links": {
          "release": {
            "href": "https://apis.danklco.com/apache-release-info/DERBY/12342992"
          },
          "jira": {
            "href": "https://issues.apache.org/jira/browse/DERBY-7118"
          }
        }
      },
      {
        "issue": "DERBY-6988",
        "title": "Tasks for producing a Derby 10.14.2 patch release",
        "type": "Task",
        "_links": {
          "release": {
            "href": "https://apis.danklco.com/apache-release-info/DERBY/12342992"
          },
          "jira": {
            "href": "https://issues.apache.org/jira/browse/DERBY-6988"
          }
        }
      }
    ]
  }
}